src/aws_encryption_sdk/identifiers.py

```diff
@@ -333,8 +334,14 @@ class ContentAADString(Enum):
 class KeyringTraceFlag(Enum):
     """KeyRing Trace actions."""
 
-    WRAPPING_KEY_GENERATED_DATA_KEY = 1
-    WRAPPING_KEY_ENCRYPTED_DATA_KEY = 1 << 1
-    WRAPPING_KEY_DECRYPTED_DATA_KEY = 1 << 2
-    WRAPPING_KEY_SIGNED_ENC_CTX = 1 << 3
-    WRAPPING_KEY_VERIFIED_ENC_CTX = 1 << 4
+    @attr.s
+    class KeyringTraceFlagValue(object):
+        """Keyring trace flags do not have defined serializable values."""
+
+        name = attr.ib()
+
+    WRAPPING_KEY_GENERATED_DATA_KEY = KeyringTraceFlagValue("WRAPPING_KEY_GENERATED_DATA_KEY")
```

Comment:
Do these need to all start with "WRAPPING_KEY"?

Reply:
I think we went with that because that's what the prevailing naming was at the time (last summer). It looks like the names in the spec are now the simpler version. I'll update to use those names instead.